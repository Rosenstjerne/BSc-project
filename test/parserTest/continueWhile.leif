var int a,b;
a = 0;
b = 5;

while (a < b) {
    a = a + 1;
    if (a == 3) {
        continue;
    }
    print(a);
}