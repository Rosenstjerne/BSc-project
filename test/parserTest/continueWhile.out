function null main() {
	var int a, b;
	
	a = 0;
	b = 5;
	while((a < b)) {
	#begin while
		a = (a + 1);
		if((a == 3))
		#begin if
			continue
			
		#end if
		print(a);
		
	}
	#end while
	
	
}
